src/lib.rs

```diff
@@ -33,12 +33,126 @@ impl<'a> Image<'a> {
             format,
         }
     }
+
+    fn get_alpha(&self, x: usize, y: usize) -> f32 {
+        self.pixels[x * 4 + 3 + y * 4 * self.width as usize] as f32 / 255.0
+    }
+
+    pub fn calculate_alpha_coverage(&self, alpha_cutoff: Option<f32>) -> f32 {
+        self.calculate_scaled_alpha_coverage(alpha_cutoff, 1.0f32)
+    }
```

Comment:
Is there a use-case to have these functions `pub` instead of private of `pub(crate)`?

Reply:
Forgot to change these back, initially I had the function require you to provide the target coverage you'd have to call the function yourself